.github/workflows/pr-build-workflow.yml

```diff
@@ -44,3 +44,8 @@ jobs:
           NO_REFERENCE_TASK: true
         with:
           arguments: checkAsciidocLinks check
+
+      - name: Publish Unit Test Results
+        uses: EnricoMi/publish-unit-test-result-action@v1
+        with:
+          files: '**/build/test-results/**/*.xml'
```

Comment:
I think capturing the gradle HTML test result rendering is much better than trying to dig through all the XML files in test-results.

Reply:
I'm trying a different action: https://github.com/marketplace/actions/publish-unit-test-results

Reply:
But I am not averse to capturing both.